### PR TITLE
Security Growler v2.2: OSX OSS Little Snitch like in Python [securitygrowler.rb]

### PR DESCRIPTION
**Pull request by BenjaminHCCarr**
#### Adding a new cask
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download securitygrowler.rb` is error-free.
- [x] `brew cask style --fix securitygrowler.rb` left no offenses.
- [X] `brew cask install securitygrowler.rb` worked successfully.
- [X] `brew cask uninstall securitygrowler.rb` worked successfully.

This is a little snitch like program that sits in the menubar and reports on port connections.

https://pirate.github.io/security-growler/

### CODE REVIEW

File: Casks/securitygrowler.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'securitygrowler' do
+  version '2.2'
+  sha256 '6e0a7a36745e1003177db4d419ec4502fb4a1ccbbf6766ba7ea8a69cad863593'
+
+  # github.com/pirate/security-growler/ was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/securitygrowler.rb]: Remove this. Only needed when hosts differ.

File: Casks/securitygrowler.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask 'securitygrowler' do
```
Comment (vitorgalvao) [inline Casks/securitygrowler.rb]: Token reference was not followed. Needs to be `security-growler`.

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

File: Casks/security-growler.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'security-growler' do
+  version '2.2'
+  sha256 '6e0a7a36745e1003177db4d419ec4502fb4a1ccbbf6766ba7ea8a69cad863593'
+  url "https://github.com/pirate/security-growler/releases/download/v#{version.major_minor}/Security.Growler.app.zip"
```
Comment (vitorgalvao) [inline Casks/security-growler.rb]: Needs a space before this. `brew cask style --fix` should fix that.

Comment (vitorgalvao) [inline Casks/security-growler.rb]: Using `#{version.major_minor}` is redundant here. Use ``#{version}`

**Comment by BenjaminHCCarr** [comment]
Hey @vitorgalvao can you take a look, I rebased and squashed. It's still failing on 10.11, but it is exiting Travis cleanly.

**Pull request merged**